content/components/page-header.mdx

```diff
@@ -304,6 +304,8 @@ The title bar can include a static page title, actions, and leading and trailing
 Pageheader allows usage of different HTML landmarks to create a more accessible UI.
 The usage of `header` and `nav` regions are allowed within PageHeader, both with different use cases.
 
+If the `PageHeader` is not contained within or contains a landmark, consider using the `role` prop to indicate one and provide it with an accessible name (perhaps the same as `PageHeader.Title`) via `aria-label`.
```

Comment:
Do you think it'd be worth providing a high-level example of the desired code here? Saying this because this is an area that engineers are often unfamiliar with, so presenting a good default to copy from might be helpful.

Reply:
yeah, good idea!